src/lib.rs

```diff
@@ -658,15 +674,25 @@ impl FromData for TableRecord {
 }
 
 #[cfg(feature = "variable-fonts")]
-const MAX_VAR_COORDS: usize = 32;
+const MAX_VAR_COORDS: usize = 64;
```

Comment:
Had to increase it because the test font has more than 32 axes. :( I hope that's okay.

Reply:
I hope so as well... Should fine I guess.